### ISSUE TITLE
Separate layer visibility and tangibility switches

### ISSUE THREAD

**Issue by Alucard648** [WohlSoft/PGE-Project#190]
Sometimes you need visible, bit not tangible layer for quick concealing of secret passages and invisible but tangible layer for setting up advanced invisible stuff, as well as Lens of Truth implementation, if those switches can be turned on/off by in-game events and scripts.

**Comment by Wohlstand** [comment]
Good idea, must be (anyway, I'll add support to toggle on/off render of the objects (if toggle render, the objects are still be active and touchable, but if hide layer, all objects are will be deactivated and will be non-touchable))
